### PR TITLE
[#IC-151] Updated dependencies for fp-ts and io-ts 2.x

### PR DESCRIPTION
**Pull request by michaeldisaro**

### CODE REVIEW

File: ValidateOtp/handler.ts [hunk 1]
```diff
@@ -139,37 +151,58 @@ export function ValidateOtpHandler(
...
         )
+      ),
+      TE.foldW<
+        IResponseErrorInternal | IResponseErrorNotFound,
+        ResponseTypes,
+        OtpValidationResponse,
+        ResponseTypes
+      >(
+        e => T.of(e),
+        s => T.of(ResponseSuccessJson(s))
```
Comment (AleDore) [inline ValidateOtp/handler.ts]: why folding here instead of using `TE.orElse` ?

Comment (michaeldisaro) [reply ValidateOtp/handler.ts]: This is just a "translation" of old logic. I'll investigate to improve it.

Comment (michaeldisaro) [reply ValidateOtp/handler.ts]: Changed to toUnion by merging your fp-ts2 porting. I kept the best of both.

**Review by AleDore — APPROVED** [review]
LGTM

**Comment by michaeldisaro** [comment]
\azp run

**Pull request merged**